File: output/ChatGPT/codenet/Java/C++/s280174843.cpp
#include <iostream>
#include <sstream>

int main() {
  int abc[3];
  std::string str;
  std::string input;
  std::getline(std::cin, str);
  std::istringstream iss(str);
  for (int i = 0; i < 3; i++) {
    iss >> input;
    abc[i] = std::stoi(input);
  }
  if (abc[0] < abc[1] && abc[1] < abc[2]) {
    std::cout << "Yes" << std::endl;
  } else {
    std::cout << "No" << std::endl;
  }
  return 0;
}

File: output/CodeGen/codenet/Java/Python/s280174843.py
def main():
    abc = [0, 0, 0]
    str = input()
    inp = str.split(" ", -1)
    for i in range(len(inp)):
        abc[i] = int(inp[i])
    if abc[0] < abc[1] and abc[1] < abc[2]:
        print("Yes")
    else:
        print("No")

if __name__ == "__main__":
    main()